Week2/homework/js-exercises/5-drinks.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+// Empty array declaration
+let drinkTray = [];
+
+// 3 drink types
+const drinkTypes = ["cola", "lemonade", "water"];
+
+/* I know this is not the way I was supposed to follow. It gives the desired result but I should have created a conditional and then
+push.I did a lot of research on it, I tried to use .filter to see the number of occurences but I didn't get it fully so
+I decided not to use it until I learn it better.*/
+
+for(let i = 0; i < 5; i++){
+   drinkTray.push(drinkTypes[i % 3]) 
+}
+console.log(drinkTray);
+console.log("Hey guys I brought a " + drinkTray + "!");
```

Comment:
👍 
True that using filter makes this exercise much easier. You will learn about this in JS2.

Another solution might be:
```
const drinkTypes = ["cola", "lemonade", "water"];
let drinkTray = [];
let currentDrinkIndex = 0;
do {
  let currentDrinkType = drinkTypes[currentDrinkIndex];
  let currentDrinkCounter = 0;
  for (let i=0 ; i<drinkTray.length ; i++) {
    if (drinkTray[i] === currentDrinkType) {
      currentDrinkCounter++;
    }
  }
  if (currentDrinkCounter < 2) {
    drinkTray.push(currentDrinkType);
  } else {
    currentDrinkIndex++;
  }
} while (drinkTray.length < 5)
console.log(drinkTray);
console.log("Hey guys I brought a " + drinkTray + "!");
```